ui/src/app/store/user/types.ts

```diff
@@ -5,12 +5,20 @@ export type User = Model & {
   completed_intro: boolean;
   email: string;
   global_permissions: string[];
+  is_local: boolean;
   is_superuser: boolean;
   last_name: string;
+  last_login: string;
+  machines_count: number;
   sshkeys_count: number;
   username: string;
 };
 
+export type UserWithPassword = User & {
```

Comment:
I might be missing something, but I'm not sure if this type should be added here. It's a type that is specific to a form and I don't think it should be used anywhere but in that form. I think normally this kind of type would just be the prop type of the form component.